packages/shared/components/layouts/website-section/default.marko

```diff
@@ -60,7 +60,7 @@ $ const { id, alias, name, pageNode } = input;
         component-input={ aliases }
         fragment-name="content-list"
         query-name="website-scheduled-content"
-        query-params={ sectionId: id, limit: 12, skip: 15, requiresImage: true }
+        query-params={ sectionId: id, limit: 12, skip: 15, withSite: false, requiresImage: true }
```

Comment:
The [website-scheduled-content](https://github.com/parameter1/base-cms/blob/master/packages/web-common/src/block-loaders/website-scheduled-content.js#L23) block loader and the underlying [websiteScheduledContent](parameter1/wordpress-diverse-community) query do not support the `withSite` parameter. Have you tested these changes?

Reply:
It's possible I just misread the query names when changing some of these. I'll go back over and check this behavior